ui/src/pages/500.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useRouter } from "next/router";
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+
+const ServerErrorPage = () => {
+  const router = useRouter();
+
+  return (
+    <div className="error-page">
+      <ExclamationCircleOutlined />
+      <h1>Something went wrong</h1>
+      <p>
+        Please try again later or{" "}
+        <a type="button" onClick={() => router.back()}>
+          go back
```

Comment:
I would make it go to home, maybe it will go back where this error has been generated